static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -55,8 +56,14 @@ export default function FeedbackItem({feedbackItem}: Props) {
                 />
               ) : null}
             </Flex>
-            <Flex align="center" gap={space(0.5)}>
-              <ProjectAvatar project={project} size={12} /> {slug}
+            <Flex gap={space(1.5)}>
+              <Flex column={false} align="center" gap={space(0.5)}>
+                <ProjectAvatar project={project} size={12} title={slug} /> {slug}
+              </Flex>
+              <Flex column={false} gap={space(1.5)}>
```

Comment:
nit: `column={false}` should be the default

```suggestion
              <Flex align="center" gap={space(0.5)}>
                <ProjectAvatar project={project} size={12} title={slug} /> {slug}
              </Flex>
              <Flex gap={space(1.5)}>
```